lib/features/campaigns/screens/poster_detail.dart

```diff
@@ -41,10 +43,45 @@ class PosterDetail extends StatelessWidget {
     }
 
     var theme = Theme.of(context);
+
+    var widgetHeight = 250.0;
+    var extraRows = <Widget>[];
+    if (poi.status != PosterStatus.removed) {
+      widgetHeight += 55;
```

Comment:
This makes total sense, but maybe still add a comment why widgets are taller for the other status. (I could see myself knowing this now and forgetting in three months.) But feel free to dismiss.